### ISSUE TITLE
SCAM DATA !!

### ISSUE THREAD

**Issue by faysalnet1** [yagiraa/dolphin-free#3]
can you explain why you upload zip file contain all data browser to you server on amazon !!!! you scam poeple or what !!

C:\Users\pc\Desktop\dolphin-free-main\dolphin-free-main\jsons\successfull_upload.json (1 résultats)
	Ligne 3:     "file": "https://anty-browser-profiles.s3.eu-central-1.amazonaws.com/aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa/aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa/aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa.datadir.zip",
  C:\Users\pc\Desktop\dolphin-free-main\dolphin-free-main\main.py (3 résultats)
	Ligne 191:         if (os.path.exists(os.path.join(os.getcwd(), 'browsers', str(browser_profile_id), f'{browser_profile_id}.datadir.zip')) == False):
	Ligne 206:     zip_path = f'browsers/{browser_profile_id}/{browser_profile_id}.datadir.zip'
	Ligne 236:         archive.save(f'browsers/{browser_profile_id}/{browser_profile_id}.datadir.zip')

**Comment by faysalnet1** [comment]
I'm not a programmer but it's weird to find external links, it's our database !!

**Comment by faysalnet1** [comment]
So explain please why you this zip exist and why uploaded to-----> [https://anty-browser-profiles.s3.eu-central](https://anty-browser-profiles.s3.eu-central-1.amazonaws.com/aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa/aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa/aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa.datadir.zip)

**Comment by APTDeser** [comment]
No, you misunderstand me. What makes you think I'm a developer? You write
to my mail, how did you get it in the first place?

пт, 2 июн. 2023 г., 02:36 faysalnet1 ***@***.***>:

**Comment by ibondarev-gsu** [comment]
Bro, this is a response template, you can write whatever you want there
